src/CommunityToolkit.Maui/Views/Popup/Popup.ios.macos.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace CommunityToolkit.Maui.Views.Popup;
 
-class NativePopup
+class Popup
```

Comment:
Let's rename it to ApplePopup

Reply:
I prefer to keep it as `Popup` for a couple reasons:
- Pedro is currently working on the Popup control, and he'll be able to leverage this code
- The file name is `Popup.ios.macos.cs` which already implies that it can only run on iOS + MacCatalyst

Reply:
we have AppleSnackbar, as for Popup, it is our own Apple implementation.
I think Pedro will create Popup.shared.cs, where it will use this native popup implementation. Let's renamre this to ApplePopup or rename AppleSnackbar to just Snackbar

Reply:
I prefer to change `AppleSnackbar` to `Snackbar` 👍

Reply:
Now that I think about it, we may have to keep it as `AppleSnackbar`. 

I purposefully made `AppleSnackbar` a nested class inside of `Snackbar.ios.macos.cs` since `AppleSnackbar` is the UIView for the iOS + MacCatalyst `Snackbar`.

Maybe `SnackbarUIView` or `SnackbarUI` is a better name for this class?

Reply:
SnackbarUIView sounds good. I will check it's necessary to  rename other classes to be consistent